conf/distro/pelux.conf

```diff
@@ -17,25 +17,15 @@ TARGET_VENDOR = "-pelux"
 
 # Add distro features
 DISTRO_FEATURES_append = " \
-                          alsa         \
-                          argp         \
-                          bluetooth    \
-                          bluez5       \
-                          largefile    \
-                          multiarch    \
-                          opengl       \
-                          pci          \
```

Comment:
pci intentionally removed?

Reply:
It is part of the default configuration.

Reply:
Ah, I did not see it since the list is not sorted.